### ISSUE TITLE
[Feature]: Double-click widget to focus Label control on property pane

### ISSUE THREAD

**Issue by jsartisan** [appsmithorg/appsmith#10064]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Currently, to change the label widget or input, we have to perform 2 steps
1. Click on the widget.
2. Focus on the label control in the property pane and then change it.

Feature request consists of double-clicking on the widget that will automatically focus on label property in property pane of the widget

### Why should this be worked on?

This feature will help save some time when the user is building the app.

**Comment by Nikhil-Nandagopal** [comment]
I don't agree with this UX. This is an unexpected hidden pattern. We should either single-click the label part of the widget and focus on the property or double-click the label and edit in place. Double click and focus on a property is hard to communicate and unexpected
